### PR TITLE
Removed StreamTip Integration

### PR DESCRIPTION
**Pull request by alazymeme**
Pull request checklist:

- [✔] `CHANGELOG.md` was updated, if applicable

### CODE REVIEW

**Comment by pajlada** [comment]
Looks good, just want to give it a try this weekend

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 ## Unversioned
 
+- Minor: Removed Pleblist StreamTip integration
```
Comment (RAnders00) [inline CHANGELOG.md]: The streamtip integration is already defunct, you just removed some remnants. I think we shouldn't add a changelog entry for this reason

**Comment by pajlada** [comment]
tested and works for me

File: pajbot/web/routes/base/pleblist.py [hunk 2]
```diff
@@ -21,12 +21,7 @@ def pleblist():
     @app.route("/pleblist/host/")
     def pleblist_host():
         return render_template(
-            "pleblist_host.html",
-            has_streamtip=False,
-            streamtip_client_id="",
```
Comment (RAnders00) [inline pajbot/web/routes/base/pleblist.py]: This is what I mean, it was already semi-removed

**Pull request merged**